examples/combined_ODNP.py

```diff
@@ -298,15 +319,16 @@
             SW_kHz=config_dict["SW_kHz"],
             ret_data=DNP_data,
         )
-        time_axis_coords[j + 1]["stop_times"] = time.time()
-    DNP_data.chunk("t", ["ph1", "t2"], [len(Ep_ph1_cyc), -1])
-    DNP_data.setaxis("ph1", Ep_ph1_cyc / 4)
-    DNP_data.setaxis("nScans", r_[0 : config_dict["nScans"]])
-    DNP_data.reorder(["ph1", "nScans", "t2"])
-    DNP_data.name(config_dict["type"])
+        time_axis_coords[j + thermal_scans]["stop_times"] = time.time()
     DNP_data.set_prop("stop_time", time.time())
     DNP_data.set_prop("postproc_type", "spincore_ODNP_v4")
     DNP_data.set_prop("acq_params", config_dict.asdict())
+    DNP_data.setaxis("nScans", r_[0 : config_dict["nScans"]])
```

Comment:
If there is only one scan, does this dimension exist? Probably it should not

Reply:
I keep this hear as at lower concentrations we apply more nScans which makes the config file and data acquisition more robust for the user

Reply:
instead of opening this file up and editing the script you simply adjust the number of scans in your active ini. One file is edited instead of touching the main ODNP script

Reply:
I think that you're missing that any processing script can detect *whether or not an nScans dimension exists* very very easily.

Reply:
I see what you're saying now

Reply:
so I leave the setaxis for nScans when it's using thermal nscans but otherwise take out the line